master/inspect.go

```diff
@@ -212,5 +213,7 @@ func (m *Master) InspectSystem() *baetyl.Inspect {
 		}
 		result.Services = append(result.Services, service)
 	}
-	return &result
+	m.infostats.Lock()
+	defer m.infostats.Unlock()
+	return json.Marshal(&result)
```

Comment:
lock is used by infostats internally, please move lock logic into infostats and prepare result in infostats.